Modules/Scripted/ShapeAnalysisToolBox/DataImporter.py

```diff
@@ -33,108 +39,324 @@ class DataImporterLogic(ScriptedLoadableModuleLogic):
   def __init__(self):
     ScriptedLoadableModuleLogic.__init__(self)
 
-    self.saveCleanedData = False
-    self.testCaseDict = {}
+    self.saveCleanData = False
+    self.labelMapDict = {}
+    self.modelDict = {}
     self.segmentationDict = {}
+    self.labelRangeInCohort = (-1, -1)
     self.topologyDict = {}
-    self.consistentTopologyDict = {}
     self.polyDataDict = {}
-
+    self.TOPOLOGY_TYPES = {
+        0: 'Circle/Torus/Mobius Strip',
```

Comment:
Introduce a constant for each one:

```
TOPOLOGY_STRIP = 0
TOPOLOGY_DISK = 1
...
```

or 

```
TOPOLOGY_STRIP_TYPE = 0
TOPOLOGY_DISK_TYPE = 1
...
```

then, you would use these constant in the code.